rides/fixtures/ride_data.json

```diff
@@ -10,7 +10,7 @@
       "mileage": 15,
       "price": 5.00,
       "total_seats": 2,
-      "departure_time": "2019-05-22 16:00:00.000000",
+      "departure_time": "2019-05-22",
```

Comment:
Made this just a Date instead of a DateTime. I figured that when someone creates a Ride, we won't have them input the exact departure time "like June 6, 2019 at 1:30pm" but just "June 6, 2019". 

Let me know what you think

Reply:
Yeah, I think it's fine if we just require a start date for departure time and leave it up to the user for now to determine